packages/client/src/__testData__/customers.ts

```diff
@@ -12,7 +12,7 @@ export const saul: Customer = {
   birthday: "2001-01-01",
   covidCertificateReleaseDate: "2021-01-01",
   covidCertificateSuspended: true,
-  category: Category.Competitive,
+  category: [Category.Competitive],
```

Comment:
I think we should change the `category` name to `categories`.
This probably involves supporting both names in type declarations.
After the migration is done  we can drop `category` from the code.